### DISCUSSION TITLE
mpv crashes when I launch SPV 4.

### DISCUSSION THREAD

**Discussion by sandrob01** [zhongfly/mpv-build#110] [Q&A]
The issue started with sinchiro build mpv v0.38.0-525-g265056fa  on Jun 23 2024 00:07:06. I tried the latest zhongfly build but same problem. I just have a black screen then it disappear.

**Comment by zhongfly** [comment]
As you said this not my build special problem, please report to https://github.com/mpv-player/mpv/issues
